SYCL/BFloat16/bfloat16_builtins.cpp

```diff
@@ -1,11 +1,11 @@
-// REQUIRES: cuda
+// RUN: %if cuda %{%clangxx -fsycl -fsycl-targets=%sycl_triple -Xsycl-target-backend --cuda-gpu-arch=sm_80 %s -o %t.out %}
+// RUN: %if cuda %{%GPU_RUN_PLACEHOLDER %t.out %}
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// TODO currently the feature isn't supported on FPGA.
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
It's unclear if we really want to run this in two configurations on CUDA platform or if some `else` is missing.

Reply:
+1. The original test uses additional flags for CUDA back-end `-Xsycl-target-backend --cuda-gpu-arch=sm_80`.

I think we should compile and run only once.

Reply:
I've just tried this:

```
$ cat ../SYCL/Basic/a.cpp
// RUN: echo %if gpu-intel-pvc %{ 1 %} %else %{ 2 %}

$ llvm-lit --param target_devices=gpu --param sycl_be=level_zero --param dpcpp_compiler=$(which clang++) --param gpu-intel-pvc=True SYCL/Basic/a.cpp  -a 2>&1 | grep 'command outpu\
t' -A1
# command output:
1

$ llvm-lit --param target_devices=gpu --param sycl_be=level_zero --param dpcpp_compiler=$(which clang++) SYCL/Basic/a.cpp  -a 2>&1 | grep 'command output' -A1
# command output:
2
```

I think you can have conditional compilation like this followed by a single RUN instance.